src/utils/confirmations.ts

```diff
@@ -0,0 +1,98 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { UserCancelledError } from '@microsoft/vscode-azext-utils';
+import vscode from 'vscode';
+import { ext } from '../extensionVariables';
+
+/**
+ * Prompts the user for a confirmation based on the configured confirmation style.
+ *
+ * @param title - The title of the confirmation dialog.
+ * @param message - The message to display in the confirmation dialog. This message will be suffixed with instructions for a specific prompt.
+ * @param expectedConfirmationWord - The word that the user must type to confirm the action when the confirmation style is set to 'Word Confirmation'.
+ * @returns A promise that resolves to a boolean indicating whether the user confirmed the action.
+ */
+export async function getConfirmationAsInSettings(
+    title: string,
+    message: string,
+    expectedConfirmationWord: string,
+): Promise<boolean> {
+    const deleteConfirmation: string | undefined = vscode.workspace
+        .getConfiguration()
+        .get<string>(ext.settingsKeys.confirmationStyle);
+
+    if (deleteConfirmation === 'Word Confirmation') {
```

Comment:
Let's define a const or an enum for this

Reply:
👍